project/Dependencies.scala

```diff
@@ -167,10 +167,10 @@ object Dependencies {
       "com.typesafe.play" %% "twirl-api" % TwirlVersion,
       "com.typesafe.slick" %% "slick" % SlickVersion,
       "com.typesafe.slick" %% "slick-hikaricp" % SlickVersion,
-      "com.zaxxer" % "HikariCP" % "2.7.4",
+      "com.zaxxer" % "HikariCP" % "2.7.5",
```

Comment:
There is HikariCP 2.7.6 already: https://github.com/brettwooldridge/HikariCP/issues/1064

Reply:
Updated

Reply:
This was just matching a transitive dependency to the whitelist. This now causes the whitelist validation to fail. Should we change back to 2.7.5, or add an explicit HikariCP dependency to force it to 2.7.6?

Reply:
I think we can add it explicitly as we have been doing in general.

Reply:
I reverted it. 2.7.6 contains a fix for something we don't use.